src/tests/multihost/ad/test_ad_misc.py

```diff
@@ -223,8 +220,8 @@ def test_0002_improved_use_negative_sid_for_sid_lookup(
             tfile.write('idmap.getnamebysid(sid)\n')
             tfile.flush()
             multihost.client[0].transport.put_file(tfile.name, '/tmp/sss_nss_idmap.py')
-        multihost.client[0].run_command('python3 /tmp/sss_nss_idmap.py', raiseonerr=True)
+        multihost.client[0].run_command('python3.6 /tmp/sss_nss_idmap.py', raiseonerr=True)
```

Comment:
Definitely a bad idea to hardcode a python minor version anywhere.

Reply:
removed it